docs/maps/maps-getting-started.asciidoc

```diff
@@ -1,129 +1,123 @@
 
-. If you haven't already, open the main menu, then click *Maps*.
-. On the maps list page, click *Create map*.
+. Open the main menu, then click *Dashboard*.
+. Click **Create new**.
+. Click **Maps**.
 . Set the time range to *Last 7 days*.
-+
-A new map is created using a base tile layer.
-+
-[role="screenshot"]
-image::maps/images/gs_create_new_map.png[]
 
-[role="xpack"]
+[float]
 [[maps-add-choropleth-layer]]
-=== Add a choropleth layer
+=== Step 2. Add a choropleth layer
 
-Now that you have a map, you'll want to add layers to it.
 The first layer you'll add is a choropleth layer to shade world countries
-by web log traffic. Darker shades symbolize countries with more web log traffic,
-and lighter shades symbolize countries with less traffic.
-
-. Click *Add layer*.
-. Select *Choropleth*.
-. From the *Layer* dropdown menu, select *World Countries*.
-. Under *Statistics source*, set *Index pattern* to *kibana_sample_data_logs*.
-. Set *Join field* to *geo.src*.
-. Click the *Add layer* button.
-. Set *Name* to `Total Requests by Country`.
-. Set *Opacity* to 50%.
-. Click *Add* under *Tooltip fields*.
-. In the popover, select *ISO 3166-1 alpha-2 code* and *name* and click *Add*.
-. Under *Fill color*, select the grey color ramp.
-. Under *Border color*, change the selected color to *white*.
-. Click *Save & close*.
+by web log traffic. Darker will shades symbolize countries with more web log traffic,
+and lighter shades will symbolize countries with less traffic.
+
+. Click **Add layer**, and then click **Choropleth**.
+
+. From the **Layer** dropdown menu, select **World Countries**.
+
+. In **Statistics source**, set:
+** **Index pattern** to **kibana_sample_data_logs**
+** **Join field** to **geo.src**
+
+. Click **Add layer**.
+
+. In **Layer settings**, set:
+
+** **Name** to `Total Requests by Country`
+** **Opacity** to 50%
+
+. Add a Tooltip field:
+
+** Select **ISO 3166-1 alpha-2 code** and **name**
+** Click **Add**
+
+. In **Layer style**, set:
+
+** **Fill color** to the grey color ramp
+** **Border color** to white
+
+. Click **Save & close**.
 +
 Your map now looks like this:
 +
 [role="screenshot"]
 image::maps/images/gs_add_cloropeth_layer.png[]
 
-[role="xpack"]
+[float]
 [[maps-add-elasticsearch-layer]]
-=== Add layers for the {es} data
-
-To avoid overwhelming the user with too much data at once, you'll add two layers for {es} data.
+=== Step 3. Add layers for the Elasticsearch data
 
-* The first layer will display individual documents.
-The layer will appear when the user zooms in the map to show smaller regions.
-* The second layer will display aggregated data that represents many documents.
-The layer will appear when the user zooms out the map to show larger amounts of the globe.
+To avoid overwhelming the user with too much data at once, you'll add two layers
+for the Elasticsearch data. The first layer will display individual documents
+when the user zooms in the maps. The second layer will
```

Comment:
Change "when the user zooms in the maps" to "when the user zooms in the map", removing "s" on "map"